node/node_handler.go

```diff
@@ -281,7 +281,10 @@ func (node *Node) PostConsensusProcessing(newBlock *types.Block) {
 	utils.GetLogInstance().Info("PostConsensusProcessing")
 	if node.Consensus.IsLeader {
 		node.BroadcastNewBlock(newBlock)
+	} else {
+		utils.GetLogInstance().Info("!!! BINGO !!!", "Reached Consensus", node.Consensus.GetConsensusID())
```

Comment:
nit: "reached consensus" is used as the key to `consensusID` which is not as straightforward as "ConsensusID" and moving "Reached Consensus" to after "Bingo"

Reply:
sure. changed.